.github/workflows/verify.yml

```diff
@@ -0,0 +1,33 @@
+name: Verify every day
```

Comment:
Rename the workflow to verify_docs.yml and change the name here as well

Reply:
done